package/network/services/openvpn/files/openvpn.init

```diff
@@ -42,6 +42,7 @@ append_params() {
 		config_get v "$s" "$p"
 		IFS="$LIST_SEP"
 		for v in $v; do
+			[ "$v" == "frames_only" ] && [ "$p" == "compress" ] && unset v && append_param "$s" "$p" && echo >> "/var/etc/openvpn-$s.conf"
```

Comment:
This is not bash, so please replace the "==" by "=".

Despite, I hope you keep on top of the operator precedence and shortcuts here ...

Reply:
@adrianschmutzler You are right. I'll change it. I think I was confused by line 46/47.

Reply:
Done.